Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml

```diff
@@ -42,14 +44,44 @@
                 </a>
             </div>
             <div class="govuk-header__content">
+                <a href="#" class="govuk-header__link govuk-header__service-name">
+                    SIRSI
+                </a>
                 <nav aria-label="Menu" class="govuk-header__navigation">
                     <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" hidden>
                         Menu
                     </button>
-                    <ul id="navigation" style="float:right;" class="govuk-header__navigation-list">
+                    <ul id="navigation" class="govuk-header__navigation-list">
                         @if (User?.Identity?.IsAuthenticated == true)
                         {
+                            var ftsService = config["FtsService"] ?? "";
+                            var showMyNotice = !string.IsNullOrEmpty(ftsService);
+
                             <li class="govuk-header__navigation-item">
+                                <a class="govuk-header__link" href="/organisation-selection">
+                                    Dashboard
+                                </a>
+                            </li>
+
+                            if (showMyNotice == true)
+                            {
+                                var urlPath = Context.Request.Path;
+                                Regex regex = new Regex(RegExPatterns.OrganisationId);
+                                Match match = regex.Match(urlPath);
+                                string guid = match.Success == true ? match.Groups[1].Value : string.Empty;
+                                
+                                if (!string.IsNullOrEmpty(guid))
+                                {
+                                    ftsService = config["FtsService"]?.Replace("an_organisation_id", guid) ?? string.Empty;
+
+                                    <li class="govuk-header__navigation-item">
+                                        <a class="govuk-header__link" href="@ftsService" target="_blank">
```

Comment:
Acceptance criteria do not say anything about opening this link in a new window. Given it's supposed to be a seamless experience to navigate between the two services, I think it makes more sense to continue in the same window.

Reply:
Done